INSTALLATION.md

```diff
@@ -18,27 +20,22 @@ Download from [website](https://git-scm.com/download/win)
 
 
 ### Install Homebrew (MacOS only)
-1. Open the terminal and copy / paste this command and hit RETURN.
+Homebrew is an easy way to manage software downloads on MacOS. You don't *have* to use it, but we recommend it. 
+
+1. First, open your terminal and run `brew -v` to determine whether you have Homebrew installed. If you get a resopnse that looks something like `Homebrew 3.1.9`, you've already got it! If you get nothing back, or an error, continue.
+2. Open the terminal and copy / paste this command and hit RETURN. Go through the prompts (you will need to grant `sudo` access).
 
-`/bin/bash -c “$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"`
+`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"`
 
 2. Validate installation by typing `brew -v` in the terminal and ensure a version number is shown.
 
 ### Install Node using NVM 
 
-This will work for both MacOS and Win10. Follow instructions on this [link](https://medium.com/@nodesource/installing-node-js-tutorial-using-nvm-5c6ff5925dd8)
+This will work for both MacOS and Win10. Follow instructions on this [link](https://medium.com/@nodesource/installing-node-js-tutorial-using-nvm-5c6ff5925dd8). Be sure to read through the whole doc to find the sections within each step relevant to you (e.g. if you're using Homebrew, when you get to Step 2 look for the section, "Install NVM with Homebrew").
 
+If you install NVM using Homebrew, make sure to read the output in terminal after you run `brew install nvm`. You will need to add a few lines to your ~/.bash_profile and perhaps complete a couple other tasks.
 
-### Install Yarn and Gatsby CLI
-- This is for both MacOS and Win10
-- Note that while this app uses npm as the package manager, yarn is required to build the [uswds](https://github.com/uswds/uswds) library.
-
-1.  Open the terminal and type `sudo npm install -global yarn` and hit RETURN.
```

Comment:
I want to say we might need to keep the installation step with yarn?

Reply:
@vim-usds I moved this to the README in the `client` directory because it's client-specific. Tried to leave only the dependencies needed for multiple parts of the system in this file.

Reply:
Nice one! Thanks!